ui/lib/core/addon/components/form-field.hbs

```diff
@@ -133,12 +133,15 @@
       @value={{get @model this.valuePath}}
       @onChange={{this.setAndBroadcast}}
       @label={{this.labelString}}
-      @labelClass="title {{if (eq @mode 'create') 'is-5' 'is-4'}}"
+      @labelClass={{unless @attr.options.shouldOnlyShowSingleRow (if (eq @mode "create") "title is-5" "title is-4")}}
```

Comment:
I changed this in my most recent PR, I recommend merging in that change so that this instead would be controlled by `attr.options.isSectionHeader`
```js
@labelClass={{if @attr.options.isSectionHeader "title is-4" "is-label"}}

```

This way the section header/form-section styling isn't reliant on `onlyShowSingleRow` which it could be we want multiple rows _and_  section header styling

Reply:
Lol I forgot I haven't merged my PR...you can just revert these changes and you'll be good!

Reply:
Thanks Claire!